src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -308,6 +308,8 @@ private static void validateGetItemKeysAndAttribute(KeysAndAttributes keysAndAtt
      */
     @Override
     public CreateTableResult createTable(CreateTableRequest createTableRequest) {
+        tableMappingFactory.validateCreateVirtualTableRequest(createTableRequest);
```

Comment:
Validating that a virtual table can be created and mapped to a physical table is now done here, before we persist the virtual table description in the repo, rather than later when CRUD on the virtual table is done.